docs/source/configuration/telemetry/instrumentation/selectors.mdx

```diff
@@ -52,20 +52,20 @@ The supergraph service is executed after query parsing but before query executio
 |--------------------|-------------|-------------------------------------------------------|-----------------------------------------------------------------------------------|
 | `operation_name`   | Yes         | `string`\|`hash`                                      | The operation name from the query                                                 |
 | `operation_kind`   | No          | `string`                                              | The operation kind from the query                                                 |
-| `query`            | Yes         | `string`\|`aliases`\|`depth`\|`height`\|`root_fields` | The graphql query                                                                 |
-| `query_variable`   | Yes         |                                                       | The name of a graphql query variable                                              |
+| `query`            | Yes         | `string`\|`aliases`\|`depth`\|`height`\|`root_fields` | The GraphQL query                                                                 |
+| `query_variable`   | Yes         |                                                       | The name of a GraphQL query variable                                              |
 | `request_header`   | Yes         |                                                       | The name of a request header                                                      |
 | `response_header`  | Yes         |                                                       | The name of a response header                                                     |
 | `is_primary_response` | No       | `true`\|`false`                                       | Boolean returning true if it's the primary response and not events like subscription events or deferred responses |
-| `response_data`    | Yes         |                                                       | Json Path into the supergraph response body data (it might impact performances)   |
-| `response_errors`  | Yes         |                                                       | Json Path into the supergraph response body errors (it might impact performances) |
+| `response_data`    | Yes         |                                                       | JSON path into the supergraph response body data (it might impact performance)   |
```

Comment:
```suggestion
| `response_data`    | Yes         |                                                       | JSON Path into the supergraph response body data (it might impact performance)   |
```